diktat-rules/src/main/kotlin/org/cqfn/diktat/ruleset/utils/AstNodeUtils.kt

```diff
@@ -737,7 +737,10 @@ fun ASTNode.calculateLineColByOffset() = buildPositionInTextLocator(text)
  *
  * @return name of the file [this] node belongs to
  */
-fun ASTNode.getFilePath(): String = getUserData(KtLint.FILE_PATH_USER_DATA_KEY).let {
+fun ASTNode.getFilePath(): String = getRootNode().let {
+    require(it.elementType == FILE) { "Root node type is not FILE" }
+    it
```

Comment:
You can use `also` instead of `let` to avoid this line

Reply:
fixed